### ISSUE TITLE
Enhancement / Show source usage

### ISSUE THREAD

**Issue by azaz44** [FrSkyRC/ETHOS-Feedback-Community#2366]
Just a suggestion for (I think, big) improvement.

#### Background

On thing I find great in Ethos (comparing to OpenTX) is better way to organize all LS, SF, curves, mixes etc. I use them heavily (more than around 20 logic switches per model, dozen of special functions and heaving mixing usage) and I find it very complex in OpenTX to figure out after some time, if some logic switch, or a curve, or anything is actually used and where.

I think in Ethos this is greatly improved, because if you stop using some weird curve, you can just delete it. You don't need this LS anymore - delete it and stuff will reorder. 

While in OpenTX you always have so many curves, so many switches, so many functions, so many flight modes, no matter if you use them or not.

Plus, in Ethos you can name switches and even put comments - super great 👍

#### Enhancement

Still, it would be super super cool, if we could select somewhere a particular source, or any other resource (LS, SF, curve, mixer) and see all its usage. Or in other words, all places which depend on it.

This would make it super easy to figure out, if this thing is used, and by what. And - of course - what are consequences of changing it.

This could be done in 2 ways:
- either just give us such a function. Maybe in a separate screen somewhere, or by holding / invoking menu on any such "resource"
- or just give us such possibilities in LUA. Then we can help ourselves and write some nice tool to do this, which will not clutter main UI with advanced stuff. (Although I think this is very useful for every user, even novice).

#### Example

![image](https://user-images.githubusercontent.com/42830563/214125451-6c1893ba-0009-49c1-8ee0-79ab3a1abed6.png)

**Comment by Jedsters** [comment]
I agree that could be useful, however wonder if it would be more useful in the Suite? That way if you want to change things you can see on a PC what depends on what  while making the changes on a model. If it is all done on the Tx then you'll have to manually make a note of everything before updating it for a model.

Also an way of identifying all orphaned resources would be nice. I have a feeling I have the odd unused curve, but short of checking dozens of mixers and outputs I can't easily tell.

**Comment by azaz44** [comment]
Why in Suite, if you make all programming on the radio? Suite does not have any option to edit model at this point.

Such function would help you with your curve. You would do "Show Usage..." on it, and you would then quickly figure out if it;s used anywhere or not. That's exactly what this feature is about.

**Comment by azaz44** [comment]
No, no stale - this would be so useful.
BTW - it would be nice to have it for switches too, and pots as well.

**Comment by azaz44** [comment]
Don't stale please, this would be super great option.

**Comment by azaz44** [comment]
Still valid.

**Comment by azaz44** [comment]
Still valid and very important.

**Comment by azaz44** [comment]
Still valid

**Comment by azaz44** [comment]
Still valid, would be a great feature
